omero/users/whatsnew.txt

```diff
@@ -3,17 +3,27 @@ What's new for OMERO 5.3 for users
 
 Updates and new features for OMERO 5.3 include:
 
-- ROI Folders which allow you to organize your ROIs into a hierarchical tree
-  structure of folders so they can be sorted e.g. by phenotypes or ontologies
+- ROI Folders in OMERO.insight which allow you to organize your ROIs into a
```

Comment:
We need to mention here that:
  - the feature is also a CLI feature (not only Insight)
  - there are limitations to this feature in Insight, see https://github.com/openmicroscopy/design/issues/16#issuecomment-287818201